mtb-pdl-cat1/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Add mtb-pdl-cat1
+zephyr_compile_definitions(${CONFIG_SOC_PART_NUMBER})
+zephyr_include_directories(drivers/include)
+zephyr_include_directories(devices/COMPONENT_CAT1A/include)
+zephyr_include_directories(devices/COMPONENT_CAT1A/include/ip)
+zephyr_include_directories(devices/COMPONENT_CAT1A/templates/COMPONENT_MTB)
+
+zephyr_library_sources(devices/COMPONENT_CAT1A/source/cy_device.c)
+zephyr_library_sources(drivers/source/TOOLCHAIN_GCC_ARM/cy_syslib_gcc.S)
+
+zephyr_library_sources_ifdef(CONFIG_CPU_CORTEX_M4
+devices/COMPONENT_CAT1A/templates/COMPONENT_MTB/COMPONENT_CM4/system_psoc6_cm4.c)
+zephyr_library_sources_ifdef(CONFIG_CPU_CORTEX_M0PLUS
+devices/COMPONENT_CAT1A/templates/COMPONENT_MTB/COMPONENT_CM0P/system_psoc6_cm0plus.c)
+
+# Peripheral drivers
+zephyr_library_sources_ifdef(CONFIG_SERIAL drivers/source/cy_scb_uart.c)
+zephyr_library_sources_ifdef(CONFIG_ADC drivers/source/cy_sar.c)
+zephyr_library_sources_ifdef(CONFIG_I2C drivers/source/cy_scb_i2c.c)
+zephyr_library_sources_ifdef(CONFIG_SPI drivers/source/cy_scb_spi.c)
+zephyr_library_sources_ifdef(CONFIG_COUNTER drivers/source/cy_tcpwm_counter.c)
+zephyr_library_sources_ifdef(CONFIG_PWM drivers/source/cy_tcpwm_pwm.c)
+zephyr_library_sources_ifdef(CONFIG_LPTIMER_INFINEON_CAT1 drivers/source/cy_mcwdt.c)
+zephyr_library_sources_ifdef(CONFIG_RTC_INFINEON_CAT1 drivers/source/cy_rtc.c)
+zephyr_library_sources_ifdef(CONFIG_SOC_FAMILY_PSOC6 drivers/source/cy_sysint.c)
+
+if(CONFIG_TRNG_INFINEON_CAT1)
```

Comment:
Where are defined the Kconfig with INFINEON_CAT1?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/44211/files#diff-aac2c98a0043e7c1c60d01b3c48623fb861aa58ca8fa7a6bb2a47d0da9dfb91c (from PR#44211)

in scope of this PR i remove any CAT1 configs, will add to PR#44211.
now we have easy handle update Cakefiles in zephyr/modules/hal_infineon (https://github.com/zephyrproject-rtos/zephyr/pull/43494/commits/dd34a7e41a9fe0869412012b0347bef040da0c6b)

Reply:
Ok. don't forget to adjust each commit properly.